react/features/video-quality/components/VideoQualityLabel.web.js

```diff
@@ -219,7 +207,6 @@ function _mapStateToProps(state) {
         _audioOnly: audioOnly,
         _conferenceStarted: Boolean(conference),
         _filmstripVisible: visible,
-        _remoteVideosVisible: shouldRemoteVideosBeVisible(state),
```

Comment:
Could you do me a big favor and also remove this logic from recording label? I had duplicated the logic for recording label reactification, with the intent to refactor later, but moving the local thumbnail up basically saves me a lot of work.